Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel.xml

```diff
@@ -19,6 +19,15 @@
     </comment>
   </info>
 
+  <materials>
+    <material name="LKr">
+      <!--<D value="1.396" unit="g/cm3" />-->
+      <D value="2.416" unit="g/cm3" />
+      <composite n="1" ref="Kr"/>
+    </material>
+  </materials>
+
```

Comment:
Can't we put this in `DetCommon/compact/materials.xml`?

Reply:
Yeah, done.